pyvista/plotting/plotting.py

```diff
@@ -197,6 +197,7 @@ def __init__(
         theme=None,
     ):
         """Initialize base plotter."""
+        super().__init__()  # cooperative multiple inheritance
```

Comment:
So for this to work I think we need to add `**kwargs` in the signature of our `__init__` (we probably don't have/want to have `*args` considering that everything we have are intended to be kwargs). And then
```suggestion
        theme=None,
        **kwargs,
    ):
        """Initialize base plotter."""
        super().__init__(**kwargs)  # cooperative multiple inheritance
```

Otherwise we're robbing `DuperType` the possibility to use additional kwargs in its `__init__`.